.appveyor.yml

```diff
@@ -17,7 +17,7 @@ install:
   - pip install tox
   - ps: Start-Process npm "run httpbin" -PassThru
   - git clone https://github.com/juj/emsdk.git
-  - cd emsdk && git checkout 9538381d56272818cba77a808ba46fdac6c961c8 && cd ..
+  - cd emsdk && git checkout 1b1f08f356d3f10d91e50c09dda6d44372ca908c && cd ..
```

Comment:
Emscripten changed their binary feed urls so we had to update to a newer version of the sdk tools that use the newer feed urls.
This does not changed the emscripten compiler version, it just changes where the compiler is downloaded from.